src/test/java/io/vertx/test/core/HttpTest.java

```diff
@@ -3179,6 +3255,7 @@ public void testTLSClientRevokedServerCert() throws Exception {
 
   @Test
   //Client specifies cert that the server does not trust via a revoked certificate of the CA
+  @Ignore // this test must be reviewed - it was "passing" when "fail" was expected
```

Comment:
I do agree with that, but i wanted the test to be reviewed by one of you, preferably the one o designed the test, so he/she can decide what to do with it. That's why i left it there (ignored though).